src/djangosaml2_spid/conf.py

```diff
@@ -332,6 +332,9 @@ def config_settings_loader(request: Optional[HttpRequest] = None) -> SPConfig:
                 # be signed. If set to True, the SP will not consume
                 # any SAML Responses that are not signed.
                 "want_assertions_signed": True,
+                "want_response_signed": settings.SAML_CONFIG.get(
+                    "want_response_signed", True
```

Comment:
Not possibile in SPID :)
this PR MUST be rejected because this feature is for tests only.

use djangosaml2 instead if you need more flexibility